js/followUnfollowUser/followUser.mjs

```diff
@@ -0,0 +1,20 @@
+import { checkFollowing } from "./checkFollowing.mjs";
+import { checkFollow } from "./index.mjs";
+
+export async function followUser(url) {
```

Comment:
Since this function handles both following and unfollowing it probably makes sense to call it something more generic.